src/main/java/network/brightspots/rcv/GuiTabulateController.java

```diff
@@ -208,13 +223,30 @@ private void watchParseCvrServiceProgress(Service<LoadedCvrData> service) {
           enableButtonsUpTo(tabulateButton);
           LoadedCvrData data = service.getValue();
           tabulateButton.setText("Tabulate " + String.format("%,d", data.numCvrs()) + " ballots");
+
+          // Populate the per-source CVR count table, and calculate the width of the filename column
+          perSourceCvrCountTable.getItems().clear();
+          int maxFilenameLength = 0;
+          for (ResultsWriter.CvrSourceData sourceData : data.getCvrSourcesData()) {
+            String countString = String.format("%,d", sourceData.getNumCvrs());
+            String fileString = new File(sourceData.source.getFilePath()).getName();
+            perSourceCvrCountTable.getItems().add(new Pair<>(fileString, countString));
+            maxFilenameLength = Math.max(maxFilenameLength, fileString.length());
+          }
+          perSourceCvrColumnFilepath.setPrefWidth(getEstWidthForStringInPixels(maxFilenameLength));
```

Comment:
> Is there any risk of the same file name in different folders? Should we show the full path? If we do, it looks even harder to read.

One way you could handle this is by making the columns resizable by the user, and having this column justify right and set to a fixed width (so it first shows them the file name, and if they expand it, it can show the path). Alternatively, you could split the filename into one column, and the path into a separate one (maybe to the right of it).

Defer to @yezr on whether or not this is worth it.

Reply:
It is currently resizeable by the user, and I tried that at first but I found that manually expanding a cell in a horizontally-scrolling window is actually kind of hard: you're expanding, but so is the container, so it scrolls -- so you're resizing controls three things: the cell size, the container size, and the scroll position. This felt clunkier than the current solution.

Reply:
Ok, sounds good. 👍